README.md

```diff
@@ -157,3 +157,8 @@ Common usage:
   ```shell
   docker run --rm -v ~/solid-config:/config -p 3000:3000 -it css:latest -c /config/my-config.json
   ```
+- If you run the Docker image on your selfhosted infrastructure, you need to supply the domainname:
+  ```shell
+  docker run --rm -p 3000:3000 -it css:latest -c /config/my-config.json -b http://example.org
+  ```
+  Note that you should not use `https` here as your reverse proxy strips that away for the server.
```

Comment:
Isn't this configuration dependent?

I don't think it is correct to say that a reverse proxy strips https away.

Reply:
Following from the discussion from #645 it probably will also break things.